src/Microsoft.DotNet.Darc/src/Darc/Helpers/LocalSettings.cs

```diff
@@ -77,70 +76,64 @@ public static LocalSettings LoadSettingsFile(CommandLineOptions options)
     public static DarcSettings GetDarcSettings(CommandLineOptions options, ILogger logger, string repoUri = null)
     {
         LocalSettings localSettings = null;
-        DarcSettings darcSettings = new DarcSettings();
-        darcSettings.GitType = GitRepoType.None;
+        DarcSettings darcSettings = new DarcSettings
+        {
+            GitType = GitRepoType.None
```

Comment:
The None value seems to be a good default. If None was the first element in the enum (see https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/enum), we wouldn't have to initialize it here, but it might be a risky change if there is already some code relying on GitHub being the default value of the enum.

Reply:
I agree but let's not change functionality in this PR.